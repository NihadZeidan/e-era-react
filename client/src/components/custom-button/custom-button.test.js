import { shallow } from "enzyme";
import CustomButton from "./custom-button.component";

it("Should render Custom Button Component", () => {
  const mockProps = {
    children: "test children",
    color: "red",
  };
  const wrapper = shallow(<CustomButton {...mockProps} />);
  expect(wrapper.props().children).toEqual("test children");
  expect(wrapper.props().color).toEqual("red");
  expect(wrapper).toMatchSnapshot();
});
