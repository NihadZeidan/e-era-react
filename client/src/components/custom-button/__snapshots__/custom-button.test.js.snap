// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Custom Button Component 1`] = `ShallowWrapper {}`;
